src/Joomlatools/Console/Command/Site/Download.php

```diff
@@ -73,6 +86,10 @@ protected function execute(InputInterface $input, OutputInterface $output)
 
         $this->versions = new Versions();
 
+        if (null !== ($val = $input->getOption('clone'))) {
```

Comment:
@msaladna I'm not sure why this if is put here. I guess this part was meant to go into the \Joomlatools\Console\Command\Site\Create class?

Reply:
Correct. I can't read diffs.